GUI/SearchTaskWindow.xaml.cs

```diff
@@ -148,6 +148,7 @@ private void UpdateFieldsFromTask(SearchTask task)
             classicSearchRadioButton.IsChecked = task.SearchType == SearchType.Classic;
             modernSearchRadioButton.IsChecked = task.SearchType == SearchType.Modern;
             nonSpecificSearchRadioButton.IsChecked = task.SearchType == SearchType.NonSpecific;
```

Comment:
I believe this assignment is useless, since line 151 now takes care of nonSpecificSearchRadioButton state